third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -577,3 +577,8 @@ $ terraform import google_container_cluster.mycluster my-gcp-project/us-east1-a/
 
 $ terraform import google_container_cluster.mycluster us-east1-a/my-cluster
 ```
+
+~> **Note:** This resource has several convenience fields that are state-only or used on creation and are not persisted by the API. If you have these fields set in config and import a cluster, you may see diffs that may or may not require actual operations against the resource. Example behavior:
```

Comment:
```suggestion
~> **Note:** This resource has several fields that control Terraform-specific behavior and aren't present in the API. If they are set in config and you import a cluster, Terraform may need to perform an update immediately after import. Some of these updates are no-ops, and some may modify your cluster.

For example:
```

Convenience fields are fields that have been pulled up to the top level from a nested object (or, more generally, up any level but almost always to the top level). These are `"virtual"` fields that don't exist in the API; `virtual` isn't a great name user-side though. How about this message?